### PR TITLE
 feat(import): You can now import the config file

### PR DESCRIPTION
**Pull request by Gwalchmei**
wait #6

### CODE REVIEW

**Review by wixyvir — APPROVED** [review]
nice

File: app/js/client/components/Application.jsx [hunk 1]
```diff
@@ -3,30 +3,39 @@
...
-
-        this.onChange = this.onChange.bind(this);
     }
 
     componentDidMount() {
+        this.loadData();
+    }
+
+    save(data) {
+        console.log(data);
```
Comment (desaintvincent) [inline app/js/client/components/Application.jsx]: console.log

File: app/js/client/components/Application.jsx [hunk 2]
```diff
@@ -3,30 +3,39 @@
...
+        axios.post('/api/elems', {data}).then(() => {
```
Comment (desaintvincent) [inline app/js/client/components/Application.jsx:22]: we can maybe put off axios

File: app/js/client/components/Application.jsx [hunk 3]
```diff
@@ -3,30 +3,39 @@
...
+            this.setState({data: response.data});
+        });
     }
 
     render() {
         return (
         <div className="app">
-            <Header onChange={this.onChange}/>
-            <BoardTabs key={this.state.keyBoard}/>
+            <Header onChange={(data) => this.save(data)}/>
```
Comment (desaintvincent) [inline app/js/client/components/Application.jsx:36]: `onChange={this.save}`

**Review by desaintvincent — APPROVED** [review]

File: app/js/client/components/Application.jsx (hunk 3 above)
Comment (Gwalchmei) [reply app/js/client/components/Application.jsx:36]: > If calling bind annoys you, there are two ways you can get around this. [...] If you aren’t using class fields syntax, you can use an arrow function in the callback

**Pull request merged**